Assignment2/kaprekarConstant.js

```diff
@@ -0,0 +1,45 @@
+let num = "2111";
+let iterationCount = 0;
```

Comment:
Globally declaring `iterationCount` can be avoided as its getting used in only one function